src/Raven.Client/Documents/Queries/TimeSeries/TimeSeriesQueryable.cs

```diff
@@ -102,20 +101,25 @@ public interface ITimeSeriesGrouping
 
     public interface ITimePeriodBuilder
     {
-        void Milliseconds(int duration);
+        ITimeSeriesAggregationExtensions Milliseconds(int duration);
 
-        void Seconds(int duration);
+        ITimeSeriesAggregationExtensions Seconds(int duration);
 
-        void Minutes(int duration);
+        ITimeSeriesAggregationExtensions Minutes(int duration);
 
-        void Hours(int duration);
+        ITimeSeriesAggregationExtensions Hours(int duration);
 
-        void Days(int duration);
+        ITimeSeriesAggregationExtensions Days(int duration);
 
-        void Months(int duration);
+        ITimeSeriesAggregationExtensions Months(int duration);
 
-        void Quarters(int duration);
+        ITimeSeriesAggregationExtensions Quarters(int duration);
 
-        void Years(int duration);
+        ITimeSeriesAggregationExtensions Years(int duration);
+    }
+
+    public interface ITimeSeriesAggregationExtensions
```

Comment:
To mach other parts of the API rename this to `ITimeSeriesAggregationOperations`

Reply:
done